### ISSUE TITLE
Project Will not Start - Lack Of Resource

### ISSUE THREAD

**Issue by TorakTu** [Spasi/LWJGL-FX#4]
I tried to run this code, I got the latest LWJGL library ( 2.9.1 ) and I even tested with the library that is included with this project, same issue on both. Project just will not start. It has 15 errors total.

This is the error I get....
http://pastebin.com/raw.php?i=NV3TjqeS

The top error is pointing to line 68. I then went to line 68 where it shows to load the image icon ( lwjgl_32x32.png ) and when I comment that out, since its only an image icon, I get down to only 1 error left....

http://pastebin.com/raw.php?i=qfJCRtP2

I'm on Windows 7 x64 with 8 Gigs Ram
Quad Core i5 2500k CPU
Nvidia GTX 660Ti Video card with 3 gigs Ram

I'm using Eclipse Version: Kepler Service Release 2
Latest SDK of Java8 is installed.

The code when setup, shows no red errors. But when i run it, the two links show you the errors that show up in the console. I am starting to think that paths are an issue here. 

Oh and by the way, I LOVE the whole idea behind this program.
LWJGL with JavaFX = Perfect !!! Thanks. :)

**Comment by Spasi** [comment]
You probably need to add the /res folder to the classpath. If that doesn't work, examine build.xml to see exactly what is required. Though, it should be simpler to use Ant to build and run.

**Comment by TorakTu** [comment]
Yes, I have tried both. I do have the res folder and I have checked the build.xml file. Nothing there that has changed this problem. I noticed that both INTEL and AMD are targeted .. could this be the problem since I am using NVidia ?
